kas-iot2050-swupdate.yml

```diff
@@ -13,3 +13,7 @@ header:
 build_system: isar
 
 target: iot2050-image-swu-example
+
+local_conf_header:
+  node-red-immutable: |
+    NODE_RED_IMMUTABLE = "1"
```

Comment:
In the current setup, node-red-gpio remains still immutable, so this is misleading.

Reply:
Not required anymore